src/comment.cpp

```diff
@@ -278,7 +280,7 @@ void CommentParser::collectParameters(const char *fulltext, FileModule *root_mod
 		// getting the node for parameter annotation
 		shared_ptr<Expression> params = CommentParser::parser(comment.c_str());
 		if (!params) {
-			params = shared_ptr<Expression>(new Literal(ValuePtr(std::string(""))));
+			params = EmptyStringLiteral;
```

Comment:
@MichaelPFrey Do you know why we assign an empty string to the annotation in cases where we don't find any annotation comments? I have a patch that removes this and regenerates the tests, but I'm not sure if it would break any UI stuff. See also #2776